tests/integrated/test-thumbnail.R

```diff
@@ -0,0 +1,88 @@
+test_that("set_thumbnail works with local images", {
+  scoped_experimental_silence()
+  img_path <- rprojroot::find_package_root_file("tests/testthat/examples/logo.png")
```

Comment:
```suggestion
  img_path <- "examples/logo.png"
```

Does this not work?

Reply:
Also, if you put this outside of a `test_that` block, you should be able to just refer to it in each one rather than having to set it each time.

Reply:
Like I replied to the other comment, I haven't futzed with these tests; they are legacy tests adapted to use the new functions.

Reply:
That to say — you're probably right, and I should check!

Reply:
This comment was in relation to old integration tests applied verbatim to new thumbnail functions. The integration tests need a more thorough look in general outside of this PR. Jon and I discussed this in an off-GitHub conversation at some point. I'm resolving this conversation to indicate that.